storage/namespace.go

```diff
@@ -234,10 +268,27 @@ func newDatabaseNamespace(
 	}
 
 	n.initShards(nopts.NeedsBootstrap())
+	go n.reportStatusLoop()
 
 	return n, nil
 }
 
+func (n *dbNamespace) reportStatusLoop() {
+	ticker := time.NewTicker(namespaceReportLoopInterval)
```

Comment:
Could you use the report interval from the instrument opts instead of using a const?

Reply:
Sure thing, can do.